openapi/index.yaml

```diff
@@ -338,25 +338,224 @@ paths:
         Note that this is an optimization for those services that need to send very high
         volumes of messages per day, to avoid having to make two API calls for each message.
       responses:
-        '200':
+        "200":
           description: Found.
           schema:
             $ref: "#/definitions/SubscriptionsFeed"
           examples:
             application/json:
               dateUTC: "2019-09-03"
-              subscriptions: [ "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" ]
-              unsubscriptions: [ ]
-        '401':
+              subscriptions:
+                [
+                  "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+                ]
+              unsubscriptions: []
+        "401":
           description: Unauthorized
-        '404':
+        "404":
           description: Subscriptions feed not available yet.
           schema:
             $ref: "#/definitions/ProblemJson"
-        '429':
+        "429":
           description: Too many requests
       parameters:
-      - $ref: "#/parameters/DateUTC"
+        - $ref: "#/parameters/DateUTC"
+  "/services":
+    post:
+      operationId: createService
+      summary: Create Service
+      description: |
+        Create a new Service with the attributes provided in the request payload.
+      parameters:
+        - in: body
+          name: body
+          description: > 
+            The service payload. It includes several attributes:
+
+            * `service_name`: `required` The name of the service to be created.
+
+            * `organization_name`: `required` The name of the organization.
+
+            * `department_name`: `required` The name of the department that refers to the organization. Generally it' s an internal department that owns the service.
+
+            * `organization_fiscal_code`: `required` The fiscal code of the organization. If it can't be provided, it should be set to a default value (i.e `"00000000000"`).
+
+            * `authorized_cidrs`: `required` An array of allowed source IPs or IP ranges for this service. It can be set to empty.
+
+            * `require_secure_channels`: `optional` If set to `true` allows only push notification, disabling email's channel. Default value is `false`.
+
+            * `service_metadata`: `optional` A set of metadata properties related to this service.
```

Comment:
I'd rather add the missing descriptions into io-functions-commons and remove this part. 

A note: the organization fiscal code will always be provided (there's not need to cite the default value)